### PR TITLE
AAP-4208 updated clickable elements

### PR DESCRIPTION
**Pull request by emurtoug**
Per [AAP-4208](https://issues.redhat.com/browse/AAP-4208), updated clickable elements to reflect proper style: 

Instead of Click **Ok** use Click btn:[Ok].

### CODE REVIEW

File: downstream/assemblies/central-auth/assembly-assign-hub-admin-permissions.adoc [hunk 1]
```diff
@@ -9,10 +9,10 @@ Hub administrative users will need to be assigned the role of _hubadmin_ in orde
 
 .Procedure
 . Navigate to the `ansible-automation-platform` realm on your SSO client.
-. Under the _Manage_ section on the side navigation bar, click *Users*.
+. Under the _Manage_ section on the side navigation bar, click btn:[Users].
```
Comment (dcdacosta) [inline downstream/assemblies/central-auth/assembly-assign-hub-admin-permissions.adoc:12]: I believe this is actually a menu item and not a button. It also violates accessibility by using a directional statement "side navigation". I suggest rewriting to something like the following:

From the navigation bars, select menu:Manage[Users].

File: downstream/assemblies/central-auth/assembly-assign-hub-admin-permissions.adoc [hunk 2]
```diff
@@ -9,10 +9,10 @@ Hub administrative users will need to be assigned the role of _hubadmin_ in orde
...
 . Select a user from the list by clicking their ID.
-. Click the *Role Mappings* tab.
+. Click the btn:[Role Mappings] tab.
```
Comment (dcdacosta) [inline downstream/assemblies/central-auth/assembly-assign-hub-admin-permissions.adoc:14]: Role Mappings is a tab selection and should be just bold with an * on either side. Unfortunately, AsciiDoctor doesn't have macros for anything other than menu and button, so any UI items that are not menus or buttons should remain simply bold.

File: downstream/assemblies/central-auth/assembly-assign-hub-admin-permissions.adoc [hunk 3]
```diff
@@ -9,10 +9,10 @@ Hub administrative users will need to be assigned the role of _hubadmin_ in orde
...
 . Using the dropdown menu under _Client Roles_, select *automation-hub*.
-. Click *hubadmin* from the _Available Roles_ field, then click *Add selected >*.
+. Click btn:[hubadmin] from the _Available Roles_ field, then click btn:[Add selected >].
```
Comment (dcdacosta) [inline downstream/assemblies/central-auth/assembly-assign-hub-admin-permissions.adoc:16]: I don't think hubadmin is a button but rather a field selection so just bold with an * on either side.

File: downstream/assemblies/central-auth/assembly-central-auth-add-user-storage.adoc [hunk 4]
```diff
@@ -9,7 +9,7 @@
 
 .Procedure
 . Log in to {AAPCentralAuth} as an SSO admin user.
-. Under the Configure section on the side navigation bar, click *User Federation*.
+. Under the Configure section on the side navigation bar, click btn:[User Federation].
```
Comment (dcdacosta) [inline downstream/assemblies/central-auth/assembly-central-auth-add-user-storage.adoc:12]: I think this is actually a menu item and violates accessibility by using a directional statement too with "side navigation". I suggest rewording to something like the following:

From the navigation bar, select menu:Configure section[User Federation].

File: downstream/assemblies/platform/assembly-installing-controller-operator.adoc [hunk 5]
```diff
@@ -33,8 +33,8 @@ When an instance of {ControllerName} is removed, the associated PVCs are not aut
 
 == Installing the {ControllerName} operator
 
-. Navigate to *Operators* > *Installed Operators*, then click on the *Ansible Automation Platform* operator.
-. Locate the *Automation controller* tab, then click *Create instance*.
+. Navigate to *Operators* > *Installed Operators*, then click on the btn:[Ansible Automation Platform] operator.
```
Comment (dcdacosta) [inline downstream/assemblies/platform/assembly-installing-controller-operator.adoc:36]: This is actually a menu selection:

Navigate to menu:Operators[Installed Operators], then click the btn:[Ansible Automation Platform] operator.

File: downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc [hunk 6]
```diff
@@ -10,10 +10,10 @@ Follow the procedures below to configure SSO with Azure Active Directory (Azure
 . In a browser, navigate to the Automation controller URL and log in using the following credentials:
   * Username: `admin`
   * Password: Use the _Administrator password_ you provided when you deployed the {PlatformNameShort} application.
-. In the Automation controller console, click *Settings* in the menu options.
-. Click *Miscellaneous System settings* under the *System* settings.
-. Click *Edit*. Enter the Automation controller URL in the Base URl of the service field.
-. Click *Save*.
+. In the Automation controller console, click btn:[Settings] in the menu options.
```
Comment (dcdacosta) [inline downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc:13]: It looks like Settings is actually a menu item and should be changed to menu:Settings[].

File: downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc [hunk 7]
```diff
@@ -10,10 +10,10 @@ Follow the procedures below to configure SSO with Azure Active Directory (Azure
...
+. Click btn:[Miscellaneous System settings] under the *System* settings.
```
Comment (dcdacosta) [inline downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc:14]: I'm not %100 certain this is a button, I'll need to check the UI once I can get access.

File: downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc [hunk 8]
```diff
@@ -24,8 +24,8 @@ To register the {ControllerName} instance in Azure, you must supply the _Azure A
 .Fetching the Azure AD OAuth2 Callback URL
 
 . In a web browser, open the {ControllerName} console.
-. Click *Settings* in the menu to open the main settings page.
-. Click *Azure AD settings* in the *Authentication* category to open the *Details* page.
+. Click btn:[Settings] in the menu to open the main settings page.
```
Comment (dcdacosta) [inline downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc:27]: This is a menu and should be menu:Settings[].

File: downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc [hunk 9]
```diff
@@ -34,14 +34,14 @@ To register the {ControllerName} instance in Azure, you must supply the _Azure A
 . Ensure that you are using the tenant where you deployed {PlatformNameShort}.
 . Type `Azure Active Directory` in the search bar.
 . Select *Azure Active Directory* from the search results.
-. Under *Manage* in the menu options, click *App registrations*.
-. In the *App registrations* page, click *+ New registration*.
+. Under *Manage* in the menu options, click btn:[App registrations].
```
Comment (dcdacosta) [inline downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc:37]: I believe this is a menu cascade selection. I suggest:

From the menu options, select menu:Manage[App registrations].

File: downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc [hunk 10]
```diff
@@ -50,8 +50,8 @@ When registration is complete, the registration page for the Automation Controll
 . In the *Automation controller application registration page* on Azure, copy and save the value of _Application (client) ID_.
 +
 You will use this value for the _Azure AD OAuth2 Key_ in the {PlatformNameShort} settings.
-. Under *Manage*, click *Certificates & secrets*.
-. Click *Client secrets* and then *+ New client secret*.
+. Under *Manage*, click btn:[Certificates & secrets].
```
Comment (dcdacosta) [inline downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc:53]: I believe these might be menu selections. Suggest rewriting to:

Select menu:Manage[Certificates & secrets].

File: downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc [hunk 11]
```diff
@@ -50,8 +50,8 @@ When registration is complete, the registration page for the Automation Controll
...
+. Click btn:[Client secrets] and then btn:[+ New client secret].
```
Comment (dcdacosta) [inline downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc:54]: I'm not sure if these are actually buttons. I think Aine works on Azure doc, can you check with her?

File: downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc [hunk 12]
```diff
@@ -69,23 +69,23 @@ Add the key _(Application (client) ID)_ and value (_Value_) of the secret that y
...
   . In *Azure AD OAuth2 Secret*, paste the secret _Value_.
-. Click *Save*.
+. Click btn:[Save].
 
 .Adding Azure Credentials to Automation controller
 
 . Open the {ControllerName} in a web browser.
-. Under *Resources*, click *Credentials*.
-. Click *Add* to open the *Create New Credentials* page.
+. Under *Resources*, click btn:[Credentials].
```
Comment (dcdacosta) [inline downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc:81]: Not sure this is a button. I think it might be a menu cascade selection. If so, suggest:

Select menu:Resources[Credentials].

You could merge steps 2 and 3 to become:
Select menu:Resources[Credentials] and click btn:[Add] to open the *Create New Credentials* page.

File: downstream/modules/aap-on-azure/proc-azure-locate-aap-marketplace.adoc [hunk 13]
```diff
@@ -6,8 +6,8 @@
 
 
 . In a browser, navigate to the Azure Marketplace.
-. Click *Private Products* from the menu on the left of the screen.
+. Click btn:[Private Products] from the menu on the left of the screen.
```
Comment (dcdacosta) [inline downstream/modules/aap-on-azure/proc-azure-locate-aap-marketplace.adoc:9]: I believe this is a menu selection and should be rewritten to:

From the navigation panel, select menu:Private Products[]

File: downstream/modules/aap-on-azure/proc-azure-update-route-tables.adoc [hunk 14]
```diff
@@ -9,13 +9,13 @@ Before you update the route tables, confirm that you satisfy the xref:proc-azure
 
 . Navigate to *Route Tables* in the Azure portal.
 . As part of your hub-and-spoke configuration, you created one or more route tables to define the routes between the networks. Click on one of these route tables.
-. From the route table menu bar, click *Routes* > *Add*.
+. From the route table menu bar, click btn:[Routes] > btn:[Add].
```
Comment (dcdacosta) [inline downstream/modules/aap-on-azure/proc-azure-update-route-tables.adoc:12]: This is a menu selection and I would rewrite to:

From the route table menu bar, select menu:Routes[Add].

File: downstream/modules/aap-on-azure/proc-azure-update-route-tables.adoc [hunk 15]
```diff
@@ -32,13 +32,13 @@ The name of the {PlatformNameShort} route table uses the following convention:
 ----
 aks-agentpool-<numbers>-routetable
 ----
-. From the route table menu bar, click *Routes* > *Add*.
+. From the route table menu bar, click btn:[Routes] > btn:[Add].
```
Comment (dcdacosta) [inline downstream/modules/aap-on-azure/proc-azure-update-route-tables.adoc:35]: This is a menu selection and I would rewrite to:

From the route table menu bar, select menu:Routes[Add].

File: downstream/modules/analytics/proc-add-roles-to-group.adoc [hunk 16]
```diff
@@ -14,7 +14,7 @@ Adding roles to an existing group will grant new permissions to existing users i
 . Click image:images/cog.png[] > *Settings*.
 . From the sidebar, navigate to *User Access* > *Groups*.
```
Comment (dcdacosta) [inline downstream/modules/analytics/proc-add-roles-to-group.adoc:15]: This should be rewritten to:

From the navigation bar, select menu:User Access[Groups].

File: downstream/modules/analytics/proc-add-roles-to-group.adoc [hunk 17]
```diff
@@ -14,7 +14,7 @@ Adding roles to an existing group will grant new permissions to existing users i
...
 . Click the group you want to add users to.
```
Comment (dcdacosta) [inline downstream/modules/analytics/proc-add-roles-to-group.adoc:16]: Minor grammar issue, suggest rewriting:

Click the group to which you want to add users.

File: downstream/modules/automation-hub/proc-add-group-to-container-repo.adoc [hunk 18]
```diff
@@ -26,7 +26,7 @@ Provide access to your container repository to users who need to work the images
 
 . Navigate to *Execution Environments*.
 . Select your container repository.
-. Click *Edit* at the top right of your window.
+. Click btn:[Edit] at the top right of your window.
```
Comment (dcdacosta) [inline downstream/modules/automation-hub/proc-add-group-to-container-repo.adoc:29]: This violates accessibility by using directional instructions with "top right of your window". I think we need to see the UI to determine how this could be rephrased to eliminate directional information.

File: downstream/titles/catalog/getting-started/master.adoc [hunk 19]
```diff
@@ -110,12 +110,12 @@ To create a *Catalog Administrator* group:
 
 
 . Navigate to *Settings* ➔ *User Access*.
```
Comment (dcdacosta) [inline downstream/titles/catalog/getting-started/master.adoc:112]: Navigate to menu:Settings[User Access].

File: downstream/titles/catalog/getting-started/master.adoc [hunk 20]
```diff
@@ -158,11 +158,11 @@ The *Catalog Administrator* can add new users to existing groups. Once added, us
 You can add users to a group in the *Groups* view in *User Access*.
 
 . Navigate to *Settings* ➔ *User Access*.
-. Click *Groups*.
+. Click btn:[Groups].
 . Select a group from the list.
-. Click *Members* and then click *Add Member*.
+. Click btn:[Members] and then click btn:[Add Member].
```
Comment (dcdacosta) [inline downstream/titles/catalog/getting-started/master.adoc:163]: I'm not %100 certain whether these are buttons but would need to see the UI to be certain.

File: downstream/titles/catalog/getting-started/master.adoc [hunk 21]
```diff
@@ -225,11 +225,11 @@ Each approval process is assigned a sequence ID that determines the priority in
 You can edit the sequence number for an approval process from the Approval process list view.
 
 . Select *Approval* from the main navigation.
-. Click the *Approval processes* tab.
+. Click the btn:[Approval processes] tab.
```
Comment (dcdacosta) [inline downstream/titles/catalog/getting-started/master.adoc:228]: Approval processes is a tab, so it should be just bold with an * on either side.

File: downstream/titles/catalog/getting-started/master.adoc [hunk 22]
```diff
@@ -266,20 +266,20 @@ See link:
...
-. Click *Save*.
+. Click btn:[Save].
 
 
 Our initial portfolio is empty, and we can start adding products from this empty state.
 
 To add a product from a platform:
 
-. Click in the *Filter by Platform* field to view connected platforms.
+. Click in the btn:[Filter by Platform] field to view connected platforms.
```
Comment (dcdacosta) [inline downstream/titles/catalog/getting-started/master.adoc:279]: Filter by Platform is a field so should stay bold with an * on either side.

File: downstream/titles/catalog/getting-started/master.adoc [hunk 23]
```diff
@@ -397,9 +397,9 @@ Once a user places an order it is created and available for review on the orders
 To approve or deny a request:
 
 . Navigate to *Approval*.
-. Click the *Requests* tab and then click on a request to expand it to detailed view.
-. Click *Approve* or *Deny*. Provide a *Reason* when denying a request.
-. Click *Submit*.
+. Click the btn:[Requests] tab and then click on a request to expand it to detailed view.
```
Comment (dcdacosta) [inline downstream/titles/catalog/getting-started/master.adoc:400]: Requests is a tab and so should remain just bold with an * on either side.

File: downstream/titles/catalog/getting-started/master.adoc [hunk 24]
```diff
@@ -409,9 +409,9 @@ Comment on requests as an administrator or approver. Comments are captured by th
 To comment on a request:
 
 . Navigate to *Approval*.
-. Click the *Requests* tab and then click on a request..
-. Click *Comment*.
-. Add a comment and click *Submit*.
+. Click the btn:[Requests] tab and then click on a request.
```
Comment (dcdacosta) [inline downstream/titles/catalog/getting-started/master.adoc:412]: Requests is a tab and should remain just bold with an * on either side.

**Review by dcdacosta — CHANGES_REQUESTED** [review]
Hi Lizzi, You did great with this because accessibility changes are more complex than they seem! I made a number of comments for these changes. It can be confusing so don't hesitate to give me a shout if you have questions.

File: downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc (hunk 11 above)
Comment (emurtoug) [reply downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc:54]: Hi @ariordan-redhat, are you able to let us know whether these elements (btn:[Client secrets] and btn:[+ New client secret]) are actually buttons? Thank you!

File: downstream/modules/automation-hub/proc-add-group-to-container-repo.adoc (hunk 18 above)
Comment (emurtoug) [reply downstream/modules/automation-hub/proc-add-group-to-container-repo.adoc:29]: Leaving this comment for now while we await review of the UI.

File: downstream/titles/catalog/getting-started/master.adoc (hunk 20 above)
Comment (emurtoug) [reply downstream/titles/catalog/getting-started/master.adoc:163]: Leaving this comment for now while we await review of the UI.

File: downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc (hunk 7 above)
Comment (emurtoug) [reply downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc:14]: Leaving this comment for now while we await review of the UI.

**Comment by ariordan-redhat** [comment]
Apart from the pending changes, looks good to me.

File: downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc (hunk 11 above)
Comment (ariordan-redhat) [reply downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc:54]: I'll take a look

Comment (ariordan-redhat) [reply downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc:54]: @emurtoug They're not buttons.
image

Comment (emurtoug) [reply downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc:54]: Thank you, @ariordan-redhat!

Comment (emurtoug) [reply downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc]: > @emurtoug They're not buttons. image

@dcdacosta I've changed this line to bold these elements instead: Click *Client secrets* and then *+ New client secret*.

File: downstream/assemblies/central-auth/assembly-central-auth-add-user-storage.adoc [hunk 25]
```diff
@@ -9,7 +9,7 @@
 
 .Procedure
 . Log in to {AAPCentralAuth} as an SSO admin user.
-. Under the Configure section on the side navigation bar, click btn:[User Federation].
+. From the  navigation bar, select menu:Configure section[User Federation].
```
Comment (dcdacosta) [inline downstream/assemblies/central-auth/assembly-central-auth-add-user-storage.adoc:12]: Looks like there is an extra space between the and navigation.

File: downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc (hunk 11 above)
Comment (dcdacosta) [reply downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc]: @emurtoug looks good. Thank you for the screen shot @ariordan-redhat, it really helps.

**Review by dcdacosta — APPROVED** [review]
Great job! I found one minor space issue. Once that's corrected and we get confirmation on whether Settings is a menu or button you're good to merge.

File: downstream/assemblies/central-auth/assembly-central-auth-add-user-storage.adoc (hunk 25 above)
Comment (emurtoug) [reply downstream/assemblies/central-auth/assembly-central-auth-add-user-storage.adoc]: Thanks for catching that! I've made that change now.
